script/Actors.sol

```diff
@@ -1,40 +1,58 @@
+/* solhint-disable one-contract-per-file */
 // SPDX-License-Identifier: BSD-3-Clause
 pragma solidity ^0.8.24;
 
 interface IActors {
     function ADMIN() external view returns (address);
-    function PROCESSOR() external view returns (address);
-    function EXECUTOR_1() external view returns (address);
     function PROPOSER_1() external view returns (address);
-}
+    function PROPOSER_2() external view returns (address);
+    function EXECUTOR_1() external view returns (address);
+    function EXECUTOR_2() external view returns (address);
 
-contract LocalActors is IActors {
-    address public constant ADMIN = address(1);
-    address public constant PROCESSOR = address(2);
-    address public constant EXECUTOR_1 = address(3);
-    address public constant PROPOSER_1 = address(4);
+    /// @dev timelock
+    function FEE_MANAGER() external view returns (address);
+    /// @dev timelock
+    function PROVIDER_MANAGER() external view returns (address);
+    /// @dev timelock
+    function BUFFER_MANAGER() external view returns (address);
+    /// @dev timelock
+    function ASSET_MANAGER() external view returns (address);
+    /// @dev timelock
+    function PROCESSOR_MANAGER() external view returns (address);
+    /// @dev multisig
+    function PAUSER() external view returns (address);
+    /// @dev multisig
+    function UNPAUSER() external view returns (address);
+    /// @dev multisig
+    function PROCESSOR() external view returns (address);
 }
 
-contract HoleskyActors is IActors {
-    address public constant ADMIN = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PROCESSOR = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant EXECUTOR_1 = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
-    address public constant PROPOSER_1 = 0x743b91CDB1C694D4F51bCDA3a4A59DcC0d02b913;
+contract TestnetActors is IActors {
```

Comment:
can we just define one address at the top YnSecurityCouncil and it to all

it being the one repeated